server/src/test/java/org/elasticsearch/cli/MultiCommandTests.java

```diff
@@ -102,4 +110,32 @@ public void testSubcommandArguments() throws Exception {
         assertFalse(output, output.contains("command1"));
         assertTrue(output, output.contains("Arguments: [foo, bar]"));
     }
+
+    public void testClose() throws Exception {
+        Command spySubCommand1 = spy(new DummySubCommand());
```

Comment:
I like mockito when it is necessary but sometimes it isn't and I feel that way here. I'd suggest having an AtomicBoolean or AtomicInteger that gets modified in the close method of a anonymous Command class. Then you can assert on that value.